macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -292,7 +309,13 @@ open class AvatarView: NSView {
 		}
 
 		initialsTextField.stringValue = AvatarView.initialsWithFallback(name: contactName, email: contactEmail)
-		avatarBackgroundColor = AvatarView.backgroundColor(for: AvatarView.colorIndex(for: contactEmail ?? contactName ?? ""))
+		updateAppearance()
```

Comment:
pass window's effective appearance?

Reply:
not passing it will default to window's effective appearance.

Reply:
Updated as per our discussion